content/en/docs/guides/image-update.md

```diff
@@ -724,6 +724,19 @@ patches:
     - op: add
       path: /spec/template/spec/containers/0/args/-
       value: --azure-autologin-for-acr
-```
+# Add this if you are using aad pod identity with managed identity (and remember to remove this line :))
```

Comment:
Comments are harmless, I don’t think we need to tell people to remove them.

Reply:
Running Kustomize build errors out because of it